client/pkg/fileutil/fileutil.go

```diff
@@ -135,3 +136,32 @@ func CheckDirPermission(dir string, perm os.FileMode) error {
 	}
 	return nil
 }
+
+// RemoveTempFile deletes useless temporary file with prefix tmp on an existing dir
+// Returns error if the dir does not exist or remove file fail
+func RemoveTempFile(dir string, lg *zap.Logger) error {
```

Comment:
Also, logger should be the first parameter, to be consistent with other functions in the project. Also, missing nil check on the logger. We should

```
if lg == nil { lg = zap.NoOp() }
```

Reply:
Done